Makefile

```diff
@@ -32,6 +32,30 @@ bin $(BIN): $(GORELEASER)
 	@echo "--- bin ---"
 	@$(GORELEASER) build --snapshot --single-target --rm-dist
 
+# Requires `wixl` from msitools https://wiki.gnome.org/msitools (or `brew install msitools`)
+# If Windows, you can download from here https://github.com/wixtoolset/wix3/releases
+WIN_BIN := dist/func-e_windows_amd64
+WIN_BIN_EXE := $(WIN_BIN)/func-e.exe
+
+$(WIN_BIN_EXE): $(GORELEASER)
+	@echo "--- win-bin ---"
+	@GOOS=windows GOARCH=amd64 $(GORELEASER) build --snapshot --single-target --rm-dist
+
+.PHONY: msi
+msi: $(WIN_BIN_EXE)
+ifeq ($(OS),Windows_NT)  # same for Windows 10 etc
+	@candle -v -arch x64 -dVersion=$(VERSION)  \
+	-dBin=$(WIN_BIN)/func-e.exe \
+	packaging/func-e.wxs
+	@light func-e.wixobj -o $(WIN_BIN)/func-e.msi -spdb
+	@rm func-e.wixobj
+else  # TODO: the exit code is 1 for some reason
```

Comment:
TODO: make sure this comment isn't lying :) I think the exit code is zero again.